packages/terra-date-input/src/DateInput.jsx

```diff
@@ -637,6 +640,7 @@ class DateInput extends React.Component {
           <option key={this.props.intl.formatMessage({ id: 'Terra.date.input.december' })} value="12">{this.props.intl.formatMessage({ id: 'Terra.date.input.december' })}</option>
         </select>
         <VisuallyHiddenText id={this.hotKeyInstructionId} text={intl.formatMessage({ id: 'Terra.date.input.hotKey' })} />
+        <VisuallyHiddenText id={this.errorMessageId} text={`, ${this.props.error}`} />
```

Comment:
we do not need visually hidden text as we already have a visual text for error message as shown in below screenshot.

<img width="1250" alt="Screenshot 2024-01-18 at 7 33 08 PM" src="https://github.com/cerner/terra-framework/assets/12869809/c0a76c48-7e8e-45fa-bb78-62e0d0d3558d">

Reply:
Updated